guides/source/active_record_multiple_databases.md

```diff
@@ -187,6 +188,46 @@ so that the file is generated in the correct directory. The command can be run l
 $ bin/rails generate migration CreateDogs name:string --database animals
 ```
 
+If you are using Rails generators, the scaffold and model generators will create the abstract
+class for you. Simply pass the database key to the command line
+
+```
+$ bin/rails generate scaffold Dog name:title --database animals
+```
+
+And a class with the database name `Animals` and `Record` will be created:
```

Comment:
I'm not sure if this is actually better, but what do you think about
```suggestion
And a -`Record` class based on the database name `animals` will be created:
```

Reply:
I incorporated all the other changes except this one because I didn't think it was clearer. Thanks for the suggestion though.

Reply:
Ended up with a different phrasing on my second pass.